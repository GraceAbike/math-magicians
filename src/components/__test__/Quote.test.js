// import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import React from 'react';
import Quote from '../quote';

it('renders correctly', () => {
  const tree = renderer

    .create(<Quote />)

    .toJSON();

  expect(tree).toMatchSnapshot();
});
