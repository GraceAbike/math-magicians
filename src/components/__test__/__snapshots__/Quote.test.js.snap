// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check any changes to the component 1`] = `
<div
  className="quote"
>
  <h2
    className="subtitle"
  >
    <strong>
      “ 
    </strong>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding.
    <strong>
       „
    </strong>
  </h2>
  <span
    className="author"
  >
    Author: William Paul Thurston
  </span>
</div>
`;

exports[`renders correctly 1`] = `
<div
  className="quote"
  data-testid="quote"
>
  <h2
    className="subtitle"
  >
    <strong>
      “ 
    </strong>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding.
    <strong>
       „
    </strong>
  </h2>
  <span
    className="author"
  >
    Author: William Paul Thurston
  </span>
</div>
`;
